lib/revision-selector/index.tsx

```diff
@@ -130,22 +150,37 @@ export class RevisionSelector extends Component<Props> {
                 onChange={this.onSelectRevision}
               />
             </div>
-            <div className="revision-buttons">
-              <button
-                className="button button-secondary button-compact"
-                onClick={this.onCancelRevision}
-              >
-                Cancel
-              </button>
-              <button
-                aria-label={`Restore revision from ${revisionDate}`}
-                disabled={isNewest}
-                className="button button-primary button-compact"
-                onClick={this.onAcceptRevision}
-              >
-                Restore
-              </button>
-            </div>
+            <section className="revision-actions">
+              <div className="revision-deleted-tags-toggle">
+                <ToggleControl
+                  id="revision-deleted-tags-checkbox"
+                  checked={showDeletedTags}
+                  onChange={toggleDeletedTags}
+                />
+                <label
+                  htmlFor="revision-deleted-tags-checkbox"
+                  className="revision-deleted-tags-toggle-label"
+                >
+                  Restore deleted tags
+                </label>
+              </div>
```

Comment:
This parts adds the toggle control as you can see in the following screenshots:

**Light theme**
Untoggled | Toggled
--|--
<img width="796" alt="Screenshot 2021-04-08 at 17 33 52" src="https://user-images.githubusercontent.com/14905380/114055566-2bf3a600-9891-11eb-9f92-0cbe31b8dddb.png">|<img width="794" alt="Screenshot 2021-04-08 at 17 34 02" src="https://user-images.githubusercontent.com/14905380/114055588-331ab400-9891-11eb-9759-23c290c88b76.png">

**Dark theme**
Untoggled | Toggled
--|--
<img width="794" alt="Screenshot 2021-04-08 at 17 34 11" src="https://user-images.githubusercontent.com/14905380/114055615-39109500-9891-11eb-96d1-33e855dd5f72.png">|<img width="793" alt="Screenshot 2021-04-08 at 17 34 17" src="https://user-images.githubusercontent.com/14905380/114055628-3c0b8580-9891-11eb-83ea-9bd0c348191c.png">

cc @SylvesterWilmott